appserver/admingui/web/src/main/resources/org/glassfish/web/admingui/Strings.properties

```diff
@@ -409,7 +409,7 @@ webSession.sessionTimeoutHelp=Maximum number of seconds that an inactive session
 webManager.reapIntervalLabel=Reap Interval:
 webManager.reapIntervalHelp=Number of seconds until inactive session data is deleted.
 webManager.maxSessionsLabel=Max Sessions:
-webManager.maxSessionsHelp=Maximum number of sessions that can be in cache.
+webManager.maxSessionsHelp=Maximum number of active sessions that can be in memory cache. A setting of -1 is for an infinite amount.
```

Comment:
Tbh I'd just remove the cache bit entirely - no point confusing people.
So: _"Maximum number of active sessions. A setting of -1 is for an infinite amount."_